unstructured_inference/inference/layoutelement.py

```diff
@@ -31,6 +31,7 @@ class LayoutElement(TextRegion):
     prob: Optional[float] = None
     image_path: Optional[str] = None
     parent: Optional[LayoutElement] = None
+    text_as_html: Optional[str] = None
```

Comment:
It was complaining about the missing `text_as_html` attribute when I added typing to `format_table_elements` method. Since we use that attribute then I think it should be included in a class. I was also wondering whether it should go inside this class or the parent `TextRegion` class.

Reply:
I am not very following this... it this coming from your test `text_as_html_attributes = [fe.text_as_html for fe in formatted_elements]`?

Reply:
No, it was coming from `models.chipper.UnstructuredChipperModel.format_table_elements` method after I added types to the function definition. If there is a reason for not including it in the class then it can stay as was, I will just use the `hasattribute` like you proposed. I can see that it is used like that in `unstructured` repo so maybe I should just stick with that? What do you think?

Reply:
I changed it to `hasattr`